### ISSUE TITLE
Scatter Chart

### ISSUE THREAD

**Issue by daniel-shuy** [apertureless/vue-chartjs#129]
Hi, I would like to suggest to either create a Scatter component, or include the steps to create one from a Line component in the documentation (it took me a really long time to figure this out, so I'm hoping this will help prevent others from running into the same trouble).

With vanilla `chart.js`, you could do:
```javascript
new Chart(ctx, {
    type: 'scatter',
    // ...
})
```
which is syntactic sugar for:
```javascript
new Chart(ctx, {
    type: 'line',
    // ...
    options: {
        scales: {
          xAxes: [{
            type: 'linear',
            position: 'bottom'
          }]
        }
    }
})
```

It would be nice if we could simply do:
```javascript
export default Scatter.extend({
    // ...
})
```
with `vue-chartjs` to create a Scatter Chart.

**Comment by daniel-shuy** [comment]
Turns out, the `type: 'scatter'` syntactic sugar actually consists of more than the options I listed above (I don't even know the full list myself, may have to dig into Chart.js' source code to find out).

I noticed that the Scatter chart also modified `tooltips.callbacks.label`.

It would be way easier to simply pass in `type: 'scatter'`, than to try and implement the options provided by it manually.

**Comment by apertureless** [comment]
If I find some time, I will get into it. Passing the `type` however would break with the consistency and would introduce a lot of confusion I guess. 

We have this some time ago with the `horizontal` Barchart, which accepted a third argument in the `renderChart` method. This however introduced also some side effects. Especially in the reactive Mixins which would not work, because in certain scenarios they call the `renderChart()` method, and would not know about the third argument.

The 

```javascript
export default Scatter.extend({
    // ...
}
```

would be imo the cleanest and most consistent way of doing it.

**Comment by daniel-shuy** [comment]
Yes I agree with you, creating a new Class is better than passing an argument `type`. I was actually talking about how to go about implementing it, sorry if I was being confusing.

**Comment by apertureless** [comment]
Added with PR #136 
Will be available with version `2.7.0`
